aws/provider.go

```diff
@@ -681,6 +681,7 @@ func Provider() terraform.ResourceProvider {
 			"aws_service_discovery_private_dns_namespace":             resourceAwsServiceDiscoveryPrivateDnsNamespace(),
 			"aws_service_discovery_public_dns_namespace":              resourceAwsServiceDiscoveryPublicDnsNamespace(),
 			"aws_service_discovery_service":                           resourceAwsServiceDiscoveryService(),
+			"aws_servicequotas_service_quota":                         resourceAwsServiceQuotasServiceQuota(),
```

Comment:
I know we generally use the AWS service name for the resource name, but this name has a bit of a stutter. What do you about just using `aws_service_quotas`?

Reply:
This is noted in the [Contributing Guide](https://github.com/terraform-providers/terraform-provider-aws/blob/master/.github/CONTRIBUTING.md#new-resource) with the following item:

> *Naming:* Resources should be named `aws_<service>_<name>` where `service` is the AWS short service name and `name` is a short, preferably single word, description of the resource. Use `_` as a separator.

What isn't included in the guide is the reasoning: 

* It helps operators and maintainers easier find resources by AWS service or configure [Customized Endpoints](https://www.terraform.io/docs/providers/aws/guides/custom-service-endpoints.html#available-endpoint-customizations) using the service name that is part of the resource name. There will be other resources with this service: https://github.com/terraform-providers/terraform-provider-aws/issues/9119
* Prevents conflicting naming issues should AWS release other "quota" related services/resources in the future
* Potentially sets up the resources for easier auto-generation in the future

We likely should include some/all of this rationale either in the contributing or maintaining guides for easier reference. I'll raise a PR for the contributing guide to start.

Reply:
See also: #9347 👍

Reply:
That makes sense. Thanks for rationale. I found the existing data sources after leaving the comment and quickly thought to rescind this comment. Especially, after thinking more about "Prevents conflicting naming issues should AWS release other "quota" related services/resources in the future". The contributing guide update looks good :+1: